source/layout.erb

```diff
@@ -31,19 +31,17 @@
       <%= partial "sidebar" %>
 
       <article>
-        <div class="chapter">
-          <h1>
-            <%= page_title %>
-            <a href="https://github.com/emberjs/guides/edit/master/source/<%= current_page.path.gsub('html', 'md') %>"
-               target="_blank" class="edit-page icon-pencil">Edit Page</a>
-          </h1>
-          <hr>
+        <h1>
+          <%= page_title %>
+          <a href="https://github.com/emberjs/guides/edit/master/source/<%= current_page.path.gsub('html', 'md') %>"
```

Comment:
@michaelrkn Looks like it gets to be a headache to not have it inside of the H1, yes.  Understood on the non-semantic nature.  Could we add an issue to clean this up later (post redesign and tutorial)?  Would probably help with SEO in the long-run ...